code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -528,6 +542,7 @@
 	var/mob/living/carbon/human/owner
 	var/list/datum/spellbook_entry/entries = list()
 	var/list/categories = list()
+	var/owner_is_apprentice = 0 // to see if the book belongs to an apprentice
```

Comment:
This seems kinda unnecessary as you can go `owner.mind.special_role` to find out if they're wizard or apprentice